extensions/amp-wordpress-embed/validator-amp-wordpress-embed.protoascii

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-wordpress-embed
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-wordpress-embed"
+    version: "1.0"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-wordpress-embed>
+  html_format: AMP
+  tag_name: "WORDPRESS-EMBED"
```

Comment:
Shouldn't this be `amp-wordpress-embed`? If not was a design decision made to make it not an AMP component but something on it's own?

Reply:
Updated using `AMP-WORDPRESS-EMBED`.